CHANGELOG.md

```diff
@@ -1,6 +1,20 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
+## \[UNRELEASED\]
+
+### Added
+
+### Fixed
+
+### Changed
+
+### Deprecated
+
+-   \[INTERNALS\] #1584 : Remove unused functions : `inline`, `subs`, `is_iterable_datatype`, `is_with_construct_datatype`, `is_pyccel_datatype`.
+-   \[INTERNALS\] #1584 : Remove unused class `ForIterator`.
+-   \[INTERNALS\] #1584 : Remove unused method `get_attribute`.
```

Comment:
Same comment as above. For clarity I would also mention the class: `ClassDef.get_attribute`.

Reply:
1c808dd